templates/generic_show_form.html.twig

```diff
@@ -683,6 +683,17 @@
                         {{ fields.htmlField('last_boot', item.fields['last_boot']|formatted_datetime(true), __('Last boot date')) }}
                      {% endif %}
 
+                     {% if barcode %}
+                        {{  fields.htmlField(
+                           'qrcode',
+                           barcode,
+                           __('Asset URL'),
+                           {
+                              'helper': __('Scan this QRCode to be redirected to the asset page from your mobile device')
+                           })
+                        }}
+                     {% endif %}
+
```

Comment:
```suggestion
                     {% if item_type in config("asset_types")  %}
                        {% set barcode %}
                           {{ call('Barcode::renderQRCode', [
                              item
                           ])|raw }}
                        {% endset %}

                        {{  fields.htmlField(
                           'qrcode',
                           barcode,
                           __('Asset URL'),
                           {
                              'helper': __('Scan this QRCode to be redirected to the asset page from your mobile device')
                           })
                        }}
                     {% endif %}

```

Reply:
The CFG_GLPI[“asset_types”] constant (give from twig function `config`)  is used to determine whether or not to display the QRCode.

this leaves the asset “showForm” unchanged